### ISSUE TITLE
It's possible change timeout value?

### ISSUE THREAD

**Issue by miquellaboria** [benzman81/homebridge-http-webhooks#136]
It's possible change timeout value?

**Comment by benzman81** [comment]
Could you be more precise?

**Comment by miquellaboria** [comment]
Yes! Sorry

I have Security System configured, but the http call always return a timeout, this call it may take more than 30 seconds.

Thank you!

**Comment by miquellaboria** [comment]
I see in the code there is a constant for the default timeout, my question is if is possible to change this value in the config file.

**Comment by benzman81** [comment]
@miquellaboria no, its not settable. Feel free to create a pull request. One note though, a longer http request results in longer waiting for an answer in HK. This means, that you will never get correct feedback from Siri, for example. Even 10s are way too long ;-)
